packages/azure-kusto-data/src/connectionBuilderBase.ts

```diff
@@ -110,6 +110,7 @@ export abstract class KustoConnectionStringBuilderBase {
     applicationCertificatePath?: string;
     public applicationNameForTracing: string | null = null;
     public userNameForTracing: string | null = null;
+    public clientVersionForTracing: string | null = null;
```

Comment:
You added it to connectionStringBuilder, now if the user sets it, it will be set in the ingest client, but not in the query client, since you didn't update it to take it from there

Reply:
dont we want it only in ingestion clients (and in the queued specifically)?